admin/database/postgres/migrations/0011.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE service (
+	id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+	org_id UUID NOT NULL REFERENCES orgs (id) ON DELETE RESTRICT,
+	name TEXT NOT NULL,
```

Comment:
Need to add a unique index `(org_id, lower(name))`, see migration for `projects` table for example